tests/__mocks__/handlers.ts

```diff
@@ -16,6 +16,20 @@ export const handlers = [
     }
     return HttpResponse.json(item);
   }),
+  //get member
+  http.get("https://api.github.com/orgs/:org/memberships/:username", ({ params: { username } }) => {
```

Comment:
Maybe it'd be better to seed these values within the database mock

Reply:
I think @gentlementlegen means to update the user entities inside the `user-get.json` so they are loaded in the db and then pull the user from the db here and return only the role